feedback.md

```diff
@@ -0,0 +1,19 @@
+# Static Site
+## What We're Looking For
+
+| Feature | Feedback    |
+| :------------- | :------------- |
+|  **Baseline** |   |
+|   Appropriate Git Usage	|   	|
+|   Answered comprehension questions	|   	|
+|   Page fully loads | |
+|   No broken links (regular or images) | |
+|   Includes at least 4 pages and styling | |
+|  **HTML** |   |
+|   Uses the high-level tags for organization: `header, footer, main` | |
+|   Appropriately using semantic tags: `section, article`, etc. | |
+|   All images include alternate text | |
+|  **CSS** |   |
+|   Using class and ID names in style declarations | |
+|   Style declarations are DRY  | |
+|  **Overall** |   |
```

Comment:
That's what we usually have - where we put the overall comments on the assignment